src/test/java/seedu/address/logic/commands/AddPatientCommandIntegrationTest.java

```diff
@@ -0,0 +1,49 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.commands.CommandTestUtil.assertNewCommandFailure;
+import static seedu.address.logic.commands.CommandTestUtil.assertNewCommandSuccess;
+import static seedu.address.testutil.TypicalDatabase.getTypicalDatabase;
+import static seedu.address.testutil.TypicalPatient.HOON_NAME;
+import static seedu.address.testutil.TypicalPatient.HOON_NRIC;
+import static seedu.address.testutil.TypicalPatient.HOON_PHONE;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.Messages;
+import seedu.address.model.NewModel;
+import seedu.address.model.NewModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.person.patient.Patient;
+import seedu.address.testutil.PatientBuilder;
+
+/**
+ * Contains integration tests (interaction with the Model) for {@code AddPatientCommand}.
```

Comment:
For standardisation, do you want JavaDoc for all CommandIntegrationTest?

Reply:
you think follow ab3 one or change all to javadoc less effort?

Reply:
standardise and change all to have javadoc